board/pluto/S45msd

```diff
@@ -9,7 +9,12 @@ img=/opt/vfat.img
 
 patch_html_page() {
 	LINUX=`uname -a | tr / -`
-	MODEL=`cat /sys/firmware/devicetree/base/model | tr / -`
+	RADIO=`dmesg | grep ad9361_probe | sed -e 's/\s\s*/\n/g' | grep -i AD936[34] | sed 's/[()]//g' | tr [a-z] [A-Z] | tr / -`
+	if [ -z "$RADIO" ] ; then
+		MODEL=`cat /sys/firmware/devicetree/base/model | tr / -`
+	else
+		MODEL=`cat /sys/firmware/devicetree/base/model | tr / - | sed "s/AD936[34]/$RADIO/"`
```

Comment:
Now model is different from the hw_model string in the libiio context attribute - not sure if this is what we want...

# cat /etc/libiio.ini 
[Context Attributes]
hw_model=Analog Devices PlutoSDR Rev.B (Z7010-AD9363)
hw_model_variant=1
hw_serial=104400b83991001807001100968f65adac

fw_version=v0.29-2-ge762-dirty
ad9361-phy,xo_correction=39999853
ad9361-phy,model=ad9363a

Reply:
where does the IIO context string come from? I can update that as well.

Reply:
https://github.com/analogdevicesinc/buildroot/blob/pluto/board/pluto/S23udc#L34

I would be careful doing this, someone (e.g MW) might use this string to match compatibility